api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9280,5 +9280,23 @@
 			<column name="sort_weight" type="double"></column>
 		</addColumn>
 	</changeSet>
+
+
+	<changeSet id="201604030003" author="vishnuraom">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">SELECT count(*) FROM drug_order WHERE dosing_type = 'SIMPLE' OR dosing_type = 'FREE_TEXT';</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Converting values in drug_order.dosing_type column(TRUNK-4845)</comment>
+		<update tableName="drug_order">
+			<column name="dosing_type" value="org.openmrs.SimpleDosingInstructions"/>
+			<where>dosing_type = 'SIMPLE'</where>
+		</update>
+		<update tableName="drug_order">
+			<column name="dosing_type" value="org.openmrs.FreeTextDosingInstructions"/>
```

Comment:
Initially we added dosing_type as an enum and was required, we went a head to add a changeset that set the default values for existing rows. Later we changed it from being an enum to being fully qualified java class names for reasons i won't go through and were legit, but we forgot to change the changeset that sets the default values for existing rows from being enum values to class names and this is meant to fix that.